.env.dev

```
@@ -41,13 +41,15 @@ ASSETS_SYNC_HISTORY_INTERVAL_IN_HOURS=21
 ASSETS_SYNC_ENABLED=true
 
 # IPFS
-IPFS_CLUSTER_ROOT=$IPFS_CLUSTER_ROOT
-IPFS_CLUSTER_USER=$IPFS_CLUSTER_USER
-IPFS_CLUSTER_PASSWORD=$IPFS_CLUSTER_PASSWORD
-IPFS_CLIENT_HOST=<PUT_YOUR_VALUE_HERE>
-IPFS_CLIENT_PORT=<PUT_YOUR_VALUE_HERE>
-IPFS_CLIENT_PROJECT_ID=<PUT_YOUR_VALUE_HERE>
-IPFS_CLIENT_PROJECT_SECRET=<PUT_YOUR_VALUE_HERE>
+IPFS_CLUSTER_ROOT # Cluster can be deployed locally with e2e/setupIpfsCluster/index.ts
```

```suggestion
IPFS_CLUSTER_ROOT_URL # Cluster can be deployed locally with e2e/setupIpfsCluster/index.ts
```
IMO this is more clear for someone who doesn't have seen the code / implementation.